soc/nordic/nrf53/Kconfig

```diff
@@ -197,7 +196,6 @@ config BOARD_ENABLE_CPUNET
 	bool "[DEPRECATED] NRF53 Network MCU is enabled at boot time"
 	select SOC_NRF53_CPUNET_ENABLE
 	select DEPRECATED
-	depends on !BT
```

Comment:
this is deprecated, why is it being changed?

Reply:
I'm trying to restore it's functionality to the commit before it was deprecated

Reply:
are you aware that this will be removed at some point because it is deprecated? Does all code ported to the new order work without any changes?

Reply:
This kconfig is not used anywhere. So it is fine we change it here or leave it.